building/gits/optee_os.rst

```diff
@@ -449,8 +450,41 @@ disable feature ``CFG_CRYPTO_GCM``:
     $ export CFG_CRYPTO_GCM=n
     $ make
 
-Configuration variables may then be used directly in ``sub.mk`` to trigger
-conditional compilation:
+When a configuration variable is introduced, the proper way to enable it by
+default is to write ``CFG_XXX ?= y`` in some ``.mk`` file. If it should be
+default disabled instead, simply do not set it to any value.
```

Comment:
I would encourage there is a default `?= n` defined somewhere. It's the place where we put some description of the newly introduced switch.

Reply:
Makes sense. I will change the recommendation.